app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -42,9 +48,27 @@ def create
         end
       end
 
+      def update
+        super
```

Comment:
Mind adding a comment here about the positioning of super for the flash message ordering, to make it clear?